.storybook/preview.tsx

```diff
@@ -195,6 +207,10 @@ const preview: Preview = {
   },
   argTypes: {
     $graphql: { target: "graphql" },
+    getChannel: { type: "function", target: "manager-api" },
+  },
+  args: {
+    getChannel: () => ({on: fn(), off: fn(), emit: action("channel.emit")}),
```

Comment:
When I use fn(), calls to emit do not show in the action tab. Also, notice that I have added `getChannel` as a global type if you try to write a play function and assert that getChannel.emit is called, you will run into some challenges.

Reply:
@kasperpeulen can you advise -- how best to use an action and assert?

Reply:
Yes, my preference is to use `fn`, and be able to assert in a play function and see that `emit` was called in the Actions tab.

Reply:
@thafryer see this thread for more info: https://chromaticqa.slack.com/archives/C02K88CKTFW/p1706212067193149

Reply:
The `fn` should already work and show to the action panel btw! @thafryer @tmeasday  
Sounds like a bug. Will check